addons/voice/org.openhab.voice.pico2wave/src/main/java/org/openhab/voice/pico2wave/internal/Pico2WaveService.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.voice.pico2wave.internal;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.eclipse.smarthome.core.audio.AudioException;
+import org.eclipse.smarthome.core.audio.AudioFormat;
+import org.eclipse.smarthome.core.audio.AudioStream;
+import org.eclipse.smarthome.core.voice.TTSException;
+import org.eclipse.smarthome.core.voice.TTSService;
+import org.eclipse.smarthome.core.voice.Voice;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Florian Schmidt
+ */
+@Component
+public class Pico2WaveService implements TTSService {

```

Comment:
Wouldn't it be more future proof to replace "Pico2Wave" everywhere with something like "PicoTTS"? It's more generic so the name would still make sense when one day you start using it with other audio formats besides WAV. The command name `pico2wave` could also differ per Linux distribution.

Reply:
I agree, I'll change the whole binding to use picotts as the name :)